### PR TITLE
add a Rakefile for database migrations

### PR DESCRIPTION
**Pull request by mpuncel**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    sh "migrate -url $(heroku run 'echo $DATABASE_URL') -path ./migrations up"
+  end
+
+  task :local do
+    database_name = "fridgly_devel"
+    sh "migrate -url postgres://localhost/#{database_name}?sslmode=disable -path ./migrations up"
+  end
+end
+
+task :setup do
```
Comment (marimiyachi) [inline Rakefile]: Put the `setup` task under the `migrate` namespace (which I think makes more logical sense) and then they can share the `database_name` var.

File: Rakefile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+namespace :db do
+  namespace :prod do
+    task :migrate do
+      sh "migrate -url $(heroku run 'echo $DATABASE_URL') -path ./migrations up"
+    end
+  end
+
+  namespace :local do
+    database_name = "fridgly_devel"
+    task :setup do
```
Comment (mpuncel) [inline Rakefile]: another thought would be to put other stuff in here, like installing the `migrate` tool

Comment (marimiyachi) [inline Rakefile]: word - i think this is good enough for  now though

**Comment by marimiyachi** [comment]
lgtm ;)

**Pull request merged**